.github/workflows/helmtagbump.yaml

```diff
@@ -71,3 +71,68 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  ui_helm_default_chart_tag:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: develop
+      - name: 'Get Previous tag'
+        id: uiprevioustag
+        uses: "broadinstitute/github-action-get-previous-tag@master"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      - name: 'Checkout datarepo-helm repo'
+        uses: actions/checkout@v2
+        with:
+          repository: 'broadinstitute/datarepo-helm'
+          token: ${{ secrets.HELM_REPO_TOKEN }}
+          path: datarepo-helm
+      - name: "ui chart default tag find and replace"
+        uses: docker://mikefarah/yq:3
+        with:
+          args: yq w -i datarepo-helm/charts/datarepo-ui/values.yaml image.tag ${{ steps.uiprevioustag.outputs.tag }}"
+      - name: "bump ui chart appVersion"
+        uses: docker://mikefarah/yq:3
+        with:
+          args: yq w -i datarepo-helm/charts/datarepo-ui/Chart.yaml appVersion ${{ steps.uiprevioustag.outputs.tag }}"
+      - name: "get new chart verion"
+        id: new_version
+        run: |
+          version=$(docker run --rm -v ${PWD}:/workdir mikefarah/yq:3 yq r datarepo-helm/charts/datarepo-ui/Chart.yaml 'version')
+          a=(`echo $version | sed 's/\./ /g'`)
+          ((a[2]++))
+          printf "increase default ui version: ${a[2]}\n\n"
+          new_version="${a[0]}.${a[1]}.${a[2]}"
+          echo "::set-output name=new_version::$new_version"
```

Comment:
Hrm, I wonder how robust this mechanism is.

Reply:
currently use it here https://github.com/broadinstitute/datarepo-umbrella-release-action/blob/master/src/main.sh